sycl/source/queue.cpp

```diff
@@ -80,6 +80,12 @@ context queue::get_context() const { return impl->get_context(); }
 
 device queue::get_device() const { return impl->get_device(); }
 
+ext::oneapi::experimental::queue_state queue::ext_oneapi_get_state() const {
```

Comment:
I should double-check the ABI tests to see if I need to add a new symbol to reflect this addition

Reply:
Indeed I did, pushed an updated now